lib/bambora/client.rb

```diff
@@ -3,6 +3,7 @@
 # Libraries
 require 'base64'
 require 'json'
+require 'cgi'
```

Comment:
Did you want to alphabetize this list? Does it make sense to separate Ruby standard library `require`'s from others?

Reply:
Yeah, I'll separate standard lib from others. I didn't alphabetize originally because load order matters for classes in this gem. But alphabetizing here makes sense.